Project/RAFinder/app/src/main/java/edu/rosehulman/rafinder/model/person/Administrator.java

```diff
@@ -3,9 +3,14 @@
 /**
  * A Residence Life administrator (an employee of the Office of Residence Life)
  */
-public class Administrator extends Employee
-{
-    public Administrator(String name, String room, String floor, String hall, String phoneNumber,String email,  String location){
-        super(name, room,floor, hall, phoneNumber, email, location);
+public class Administrator extends Employee {
+    public Administrator(String name,
+                         String email,
+                         String hall,
+                         String phoneNumber,
+                         int room,
+                         String status,
+                         String statusDetail) {
+        super(name, email, 0, hall, phoneNumber, room, status, statusDetail);
```

Comment:
Actually, may need to have special handling of admins, since they don't technically have a floor.